src/screens/SignTransactionSheet.tsx

```diff
@@ -765,7 +822,7 @@ export const SignTransactionSheet = () => {
     if (accountInfo.isHardwareWallet) {
       navigate(Routes.HARDWARE_WALLET_TX_NAVIGATOR, { submit: onPressSend });
     } else {
-      await onPressSend();
+      onPressSend();
```

Comment:
checking if you wanted to remove this `await` - would it be more fair to make sure that the performance tracking for `submitFn` includes the completion of `onPressSend` ?

Reply:
I wanted to separate the actual time to call the CTA instead of waiting for the inner functions to complete. Technically I think this is more of what we want?

If we await, we'll be awaiting the entire txn process.

Reply:
We could do that, just need to add other operations to track that "other stuff" part